gui/web/src/App.js

```diff
@@ -232,21 +239,21 @@ class App extends Component {
       <div>
         <div>{banner}</div>
         <Router>
-          <Header version={this.state.version}/>
+          <Header version={this.state.version} />
           <Route exact path="/"
-            render={(props) => <Bots {...props} baseUrl={baseUrl} activeError={this.state.active_error} setActiveError={this.setActiveBotError} hideActiveError={this.hideActiveError} addError={this.addError} removeError={removeBotError} getErrors={getBotErrors}/>}
-            />
+            render={(props) => <Bots {...props} baseUrl={baseUrl} activeError={this.state.active_error} setActiveError={this.setActiveBotError} hideActiveError={this.hideActiveError} addError={this.addError} removeError={removeBotError} getErrors={getBotErrors} />}
```

Comment:
can you update your git settings to ignore whitespaces?
not sure if there's a standardized way to do this but this is a [good starting point](https://stackoverflow.com/questions/3515597/add-only-non-whitespace-changes)